### DISCUSSION TITLE
The Hive integration with SecurityOnion 2.3.21

### DISCUSSION THREAD

**Discussion by Gkavach** [Security-Onion-Solutions/securityonion#9835] [General]
I need help in The hive Integrating with securityonion. I have tried but i didn't. Can you tell me the external way to integrate the hive  with SO.

**Comment by cm-ops** [comment]
We don't support `TheHive`. If you are looking for a something to send data, you could look at a SOC customization https://docs.securityonion.net/en/latest/soc-customization.html?#action-menu
